bootstrap_exercises.py

```diff
@@ -1,13 +1,14 @@
 import json
 import models
+import workout
 
-
-session = models.Session()
+session = workout.Session()
 
 exercises = json.load(open('exercises.json', 'r'))
 
 for category in exercises['exercise_categories']:
     session.add(models.ExerciseCategory(**category))
+session.commit()
```

Comment:
there's already a call to `session.commit()` here: https://github.com/dseisun/nerdouts/blob/master/bootstrap_exercises.py#L15

is this one needed to save the `ExerciseCategory` table?